static_src/components/users_selector.jsx

```diff
@@ -11,6 +11,8 @@ import { Form, FormSelect } from './form';
 import PanelDocumentation from './panel_documentation.jsx';
 import userActions from '../actions/user_actions';
 import { validateString } from '../util/validators';
+import SpaceStore from '../stores/space_store.js';
```

Comment:
Sorry for the drive-by comment, I happened to be creeping by the PR list....

The parent component of `UsersSelector` already imports both the User and Space stores, maybe we can leverage that to pass the data this component needs as props? The `stateSetter` pattern should really only be used when there isn't a clear way to get props down from a parent component